packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -1019,7 +1092,7 @@ export class ListBase extends CollectionWidget<ListBaseProperties> {
     this.setAria(groupHeaderAria, $groupBody);
   }
 
-  _renderGroup(index, group) {
+  _renderGroup(index: number, group: TItem & { template: Template }): void {
```

Comment:
Kindly check this type 'TItem & { template: Template }' after generic type removing